ops/model.py

```diff
@@ -985,6 +1006,52 @@ def location(self):
         return self._location
 
 
+class Container:
+    """Represents a named (Kubernetes) container in an application.
```

Comment:
s/Kubernetes//

Also, I believe this is related to the unit rather than application?